microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+   * [ICoreWebView2CustomSchemeRegistration::SchemeName property (get)](#)<!--no put-->
+   * [ICoreWebView2CustomSchemeRegistration::TreatAsSecure property (get](#), [put)](#)
+
+---
+
+<!-- ====================================================================== -->
+
+## 1.0.1671-prerelease
+
+Release Date: Feb 13, 2023  
+
+[NuGet package for WebView2 SDK 1.0.1671](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1671-prerelease)  
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 112.0.1671.0 or higher. 
+
+### General
+
+#### Experimental features 
+
+*  Added support for the Experimental File API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalFile interface](#)
+   * [ICoreWebView2ExperimentalFile::get_Path method](#)
+* [ICoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+---
+
+*  Added support for Experimental Object Collection View API
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+---
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:
+
+*  The SharedBuffer API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2_17 interface](#)
+	* [CoreWebView2_17::PostSharedBufferToScript method](#)
+* [CoreWebView2Environment12 interface](#)
+	* [ICoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [CoreWebView2Frame4 interface](#)
+	* [CoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [CoreWebView2SharedBuffer interface](#)
+	* [CoreWebView2SharedBuffer:: OpenStream method](#)
+	* [CoreWebView2SharedBuffer:: Close method](#)
+	* [CoreWebView2SharedBuffer:: get_Size method](#)
+	* [CoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [CoreWebView2SharedBuffer:: get_FileMappingHandle method](#)

```

Comment:
@peiche-jessica - does .NET or WinRT have this?
CoreWebView2SharedBuffer.FileMappingHandle Property